app/models/measure.rb

```diff
@@ -49,10 +67,12 @@ def different_parent
     end
   end
 
-  def parent_id_allowed_by_measuretype
-    if parent_id && !parent.measuretype&.has_parent
-      errors.add(:parent_id, "is not allowed for this measuretype")
-    end
+  def notify?
+    task? &&
+      notifications? &&
+      (!draft? && !saved_change_to_attribute?(:draft)) &&
+      (!is_archive || saved_change_to_attribute?(:is_archive)) &&
```

Comment:
should this be `is_archive?` (with the '?')

Reply:
```suggestion
      (!is_archive? || saved_change_to_attribute?(:is_archive)) &&
```

Yes it should be